R/data.table.R

```diff
@@ -1461,6 +1461,7 @@ replace_order = function(isub, verbose, env) {
   lockBinding(".GRP",SDenv)
   lockBinding(".iSD",SDenv)
 
+  #browser()
```

Comment:
don't forget about `git diff --cached` before commit, or `git log -1 -p` after commit, before push, so `git commit --amend` can be still made locally before push.